### DISCUSSION TITLE
publii theme image displays on preview but won't display on website

### DISCUSSION THREAD

**Discussion by dma2386** [GetPublii/Publii#1570] [Help]
publii theme table with image displays on preview but won't display on website.
HTML code for an image displays correctly when inserted into all other website post previews and on the uploaded website posts, but when inserted into the theme only displays correctly in the preview but doesn't display on the uploaded website.
Grateful for any help to resolve this issue, code here...
<table style="border-collapse: collapse; width: 100%;" border="1">
<tbody>
<tr>
<td style="width: 20%;"><img class="post__image post__image--right" src="file:///C:/Users/Don/Documents/Publii/sites/sleep/input/media/posts/6/UKCA-black-fill.jpg" alt="" width="100" height="100"></td>
<td style="width: 80%;">
<h5 class="align-left"><em>UK Conformity Assessed</em></h5>
<h5 class="align-left"><em>A Class I Medical Device registered at the Medicines &amp; Healthcare products Regulatory Agency and conforms with the requirements of the Medical Devices Regulations. It is an implicit declaration that the product complies with all UK laws and applicable British conformity assessment procedures.</em></h5>
</td>
<td style="width: 100%;" colspan="2"> </td>
</tr>
</tbody>
</table>

**Comment by bobmitro** [comment]
paste the source in the code box

**Reply by dma2386** [reply]
I've pasted the table code which contains the image source into the Source Code of the Hero Section.

In preview, the text appears OK in the right table cell and the UK CA image appears OK in the left table cell.

However, after it’s been uploaded, the text appears OK in the right table column BUT the UK CA image *doesn’t* appear in the left table column.

On other posts (pages) this works OK – Image on left, Text on right – after the same image source has been inserted into the left table cell using the Insert / Edit Image button.

However, Insert/Edit Image isn’t available when editing the Hero Section.

It seems that using an image source within a table doesn't work, so is there a way of using Insert/Edit with the Hero Section to get the image to appear after upload?

**Reply by dma2386** [reply]
The only way I can get the UK CA to appear in the table is to use this code, but it just appears as text, nmot as the official UKCA logo:
<tr>
<td style="width: 20%;">
<h1><strong>CA</strong></h1>
<h1><strong>UK</strong></h1>
</td>

**Reply by candidexmedia** [reply]
Would you mind sharing the live page link so that it can be inspected?

**Comment by dma2386** [comment]
Here's the code for the two text lines...
![UKCA code01](https://github.com/GetPublii/Publii/assets/70143432/5780d116-f920-4bc8-a0a9-245ef7ee646e)

**Reply by candidexmedia** [reply]
Sidenote: to properly format codeblocks in GitHub, use ` ``` ` at the top and bottom of the code. You can even specify the code language for highlighting in the top ` ``` `

For example:

 ```
| ```html
| paste code here
| ```
 ```
